src/main/java/com/github/onsdigital/logging/v2/event/BaseEvent.java

```diff
@@ -118,6 +121,11 @@ public T exception(Throwable t) {
         return (T) this;
     }
 
+    public T exceptionAll(Throwable t) {
+        this.errors = new Errors(t);
```

Comment:
In some functions, we are checking if the throwable exists and others not. Is it possible to call the function without provided the parameter? Is it worth checking here that the value is set before giving it to the Errors constructor?